models/src/main/java/com/vimeo/networking2/Video.kt

```diff
@@ -13,240 +13,169 @@ import java.util.Date
 
 /**
  * Video data.
+ *
+ * @param badge Information for the video's badge.
+ * @param categories The categories to which this video belongs.
+ * @param contentRating The content ratings of this video.
+ * @param context The context of the video's subscription, if this video is part of a subscription.
+ * @param createdTime The time in ISO 8601 format when the video was created.
+ * @param description A brief explanation of the video's content.
+ * @param download A list of downloadable files.
+ * @param duration The video's duration in seconds.
+ * @param embed Information about embedding this video.
+ * @param height The video's height in pixels.
+ * @param language The video's primary language.
+ * @param lastUserActionEventDate The time in ISO 8601 format when the user last modified the video.
+ * @param license The Creative Commons license used for the video. See [Video.licenseType].
+ * @param link The link to the video.
+ * @param live Live playback information.
+ * @param metadata The video's metadata.
+ * @param modifiedTime The time in ISO 8601 format when the video metadata was last modified.
+ * @param name The video's title.
+ * @param parentFolder Information about the folder that contains the video, or null if it is in the root directory.
+ * @param password The privacy-enabled password to watch this video. This data requires a bearer token with the private
+ * scope.
+ * @param pictures The active picture for this video.
+ * @param play The Play representation.
+ * @param privacy The video's privacy setting.
+ * @param releaseTime The time in ISO 8601 format when the video was released.
+ * @param resourceKey The resource key string of the video.
+ * @param reviewPage Information about the review page associated with this video. This data requires a bearer token
+ * with the private scope.
+ * @param fileTransferPage Information about the file transfer page associated with this video. This data requires a
```

Comment:
Optional: All other fields are alphabetized, could also shuffle order of `fileTransferPage`, `editSession`, and `isPlayable` so they match.